test/extensions/filters/http/geoip/geoip_filter_integration_test.cc

```diff
@@ -77,6 +77,28 @@ const std::string ConfigIspAndAsn = R"EOF(
         asn_db_path: "{{ test_rundir }}/test/extensions/geoip_providers/maxmind/test_data/GeoLite2-ASN-Test.mmdb"
   )EOF";
 
+const std::string ConfigIspAndAsnNotUseAsn = R"EOF(
```

Comment:
nit: the intent in this variable name sounds confusing, if we follow the same style of naming as previous configs it could be smth like "ConfigCityAndIsp" and then test name will describe the expectation on this config